src/pages/GetStarted/CustomWorkspaceTab/StorageType/StorageType.tsx

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import React from 'react';
+import {
+  FormGroup,
+  Button,
+  Modal,
+  ModalVariant,
+  Select,
+  SelectOption,
+  SelectVariant,
+  SelectDirection,
+  TextContent,
+  Text
+} from '@patternfly/react-core';
+import { connect } from 'react-redux';
+import { AppState } from '../../../../store';
+import * as WorkspaceStore from '../../../../store/Workspaces';
+import * as BrandingStore from '../../../../store/Branding';
+
+import styles from './StorageType.module.css';
+
+export enum StorageType {
+  async = 'Asynchronous',
+  ephemeral = 'Ephemeral',
+  persistent = 'Persistent',
+}
+
+type Props = {
+  storageType?: StorageType;
+  onChange?: (storageType: StorageType) => void;
+} & {
+  brandingStore: BrandingStore.State;
+  workspaces: WorkspaceStore.WorkspacesState,
+}
+type State = {
+  isOpen?: boolean;
+  selected?: string;
+  isModalOpen?: boolean;
+};
+
+export class StorageTypeFormGroup extends React.PureComponent<Props, State> {
+  storageTypes: StorageType[] = [];
+  preferredType: string;
+  options: {
+    value: string;
+    isPlaceholder?: boolean;
```

Comment:
why do you need this field? and I can't find where do you assign its value?

Reply:
```
options: string;
```

done